README.md

```diff
@@ -6,21 +6,23 @@ Numba compatible SCFG (Structured Control Flow Graph)  utilities.
 
 This repository contains Numba compatible utilities for working with SCFG
 (Structured Control FLow Graphs). SCFGs are a type of Intermediary
-Representation (IR) suitable for regularizing Python source- and bytecode.
+Representation (IR) suitable for regularizing Python source and bytecode.
 
 The code in this repository is an implementation of the CFG restructuring
 algorithms in Bahmann 2015, specifically those from section 4.1 and 4.2: namely
 "loop restructuring" and "branch restructuring". These are interesting for
-Numba because they serve to clearly identify regions within the Python source-
+Numba because they serve to clearly identify regions within the Python source
 and bytecode.
 
-Note: The project was previously known and `numba-rvsdg` and was renamed to
+Note: The project was previously known as `numba-rvsdg` and was renamed to
 `numba-scfg` in early 2025. The original scope was to implement Regional Value
 State Dependence Graphs (RVSDG), where SCFGs are necessary intermediary for the
 transformation from Python to RVSDG. But over time it became evident that the
-full RVSDG representation wasn't needed and so this package implements the SCFG
-transformations. Thus, the decision was taken to rename the package such that
-it better describes what it is about.
+full RVSDG representation was not as important as initially anticipated and so
```

Comment:
> "not as important"

That's not true. It's just being done in a later stage of the pipeline. I have the RVSDG-ization in SealIR.  Here, you can flip the emphasis to say that SCFG is an important and reusable transformation by itself to be a separate package.

Reply:
Fixed in 40167cd